### PR TITLE
Navigating links

### PR DESCRIPTION
**Pull request by Zemurin**

### CODE REVIEW

File: ProvinceMapper/Source/LinkMapper/LinkMapper.cpp [hunk 1]
```diff
@@ -38,3 +37,21 @@ void LinkMapper::exportMappings(const std::string& linksFile) const
...
+		activeVersion->deactivateLink();
+}
+
+void LinkMapper::activateLinkByIndex(const int row)
+{
+	if (activeVersion)
+		activeVersion->activateLinkByIndex(row);
+}
+
+void LinkMapper::activateLinkByID(const int ID)
```
Comment (IhateTrains) [inline ProvinceMapper/Source/LinkMapper/LinkMapper.cpp:53]: I believe ID should be unsigned if we don't expect negative IDs, unless you prefer int for negative sentinels.

File: ProvinceMapper/Source/Provinces/Province.h [hunk 2]
```diff
@@ -3,17 +3,7 @@
...
-	int y = 0;
-	unsigned char r = 0; // these colors vary depending on the state of the province.
-	unsigned char g = 0;
-	unsigned char b = 0;
-};
-
+struct Pixel;
 struct Province
 {
 	Province(int theID, unsigned char tr, unsigned char tg, unsigned char tb, std::string theName);
```
Comment (IhateTrains) [inline ProvinceMapper/Source/Provinces/Province.h:9]: I believe ID should be unsigned if we don't expect negative IDs, unless you prefer int for negative sentinels.

Comment (Zemurin) [reply ProvinceMapper/Source/Provinces/Province.h:9]: Naw, I'm the one assigning those links, and unless we start reading lines in reverse, there shouldn't be an issue.
Tis funny though, related. I wanted to make coords to offset foolproof by requiring positive heights and widths and then realized wxWidgets doesn't even bother and is giving me regular ints for coordinates. C'est la vie.

Comment (Zemurin) [reply ProvinceMapper/Source/Provinces/Province.h:9]: Oh, you meant province IDs, those are read from definition.csv. There's no negative stuff in there or the game would have some funny issues.

**Pull request merged**